src/core/qgsproject.h

```diff
@@ -72,6 +72,14 @@ class CORE_EXPORT QgsProject : public QObject
 
   public:
 
+    //! describes the type of an error
+    enum ERROR_CODE
+    {
+      NoError,
+      IOError,       // data could not be read/written
+      ParseError,    // data could not be parsed
```

Comment:
The code names look good to me, but the enum should be CamelCase to conform with the rest.